src/main/java/com/cdancy/bitbucket/rest/features/PullRequestApi.java

```diff
@@ -157,4 +162,50 @@ CommitPage commits(@PathParam("project") String project,
                                 @Nullable @QueryParam("withCounts") Boolean withCounts,
                                 @Nullable @QueryParam("limit") Integer limit,
                                 @Nullable @QueryParam("start") Integer start);
+
+    @Named("pull-request:list-activities")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/latest/bitbucket-rest.html#idm45888278197104"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/activities")
+    @Fallback(BitbucketFallbacks.ActivitiesPageOnError.class)
```

Comment:
Lets statically import this class so that we can do `@Fallback(ActivitiesPageOnError.class)`